rubailo_s/lesson9/homework.rb

```diff
@@ -0,0 +1,50 @@
+require_relative "./classes/person"
+require_relative "./classes/deck"
+
+players = [Person.new('John', 'john@test.rs', '777-7777-777'), Person.new('Mike', 'mike@test.rs', '444-4444-444')]
+
+players[0].greet(players[1])
+players[1].greet(players[0])
+
+first_player = players.sample
```

Comment:
```rb
first_player, second_player = players.shuffle
```

Reply:
Yes, thank you. This way is simpler.
Changes were added to branch.